Lib/test/test_fileinput.py

```diff
@@ -39,7 +41,7 @@ class BaseTests:
     # temp file's name.
     def writeTmp(self, content, *, mode='w'):  # opening in text mode is the default
         fd, name = tempfile.mkstemp()
-        self.addCleanup(support.unlink, name)
+        self.addCleanup(safe_unlink, name)
```

Comment:
I would prefer to use os_helper.unlink here, rather than changing the function name.

Reply:
done.
I use safe_unlink in here's  reason is that test_fileinput have import unlink as safe_unlink. 
if I import os_helper again, the imports looks like more duplicated~

Reply:
It's common to import a module and import some symbols of the same module. "Inconsistencies" are common, especially in tests which are old :-)

Reply:
For such PR which basically updates tests to point to new submodules, I prefer to leave the tests unchanged, don't change the coding style in the same PR.

Reply:
(Also, PR which only changes coding style are usually rejected. In short, the coding style cannot be updated in Python code base :-))

Reply:
> (Also, PR which only changes coding style are usually rejected. In short, the coding style cannot be updated in Python code base :-))

Got it, Thanks. I have know the pain of code churn( not easy for finding the historical  record ;-) )